ember-flight-icons/public/icons/_catalog.json

```diff
@@ -1 +1 @@
e":24},{"Name":"trend-down-16","Size":16},{"Name":"trend-up-24","Size":24},{"Name":"trend-up-16","Size":16},{"Name":"triangle-24","Size":24},{"Name":"triangle-16","Size":16},{"Name":"triangle-fill-24","Size":24},{"Name":"triangle-fill-16","Size":16},{"Name":"truck-24","Size":24},{"Name":"truck-16","Size":16},{"Name":"tv-24","Size":24},{"Name":"tv-16","Size":16},{"Name":"type-24","Size":24},{"Name":"type-16","Size":16},{"Name":"unfold-open-24","Size":24},{"Name":"unfold-open-16","Size":16},{"Name":"unfold-close-24","Size":24},{"Name":"unfold-close-16","Size":16},{"Name":"unlock-24","Size":24},{"Name":"unlock-16","Size":16},{"Name":"upload-24","Size":24},{"Name":"upload-16","Size":16},{"Name":"user-24","Size":24},{"Name":"user-16","Size":16},{"Name":"user-check-24","Size":24},{"Name":"user-check-16","Size":16},{"Name":"user-circle-24","Size":24},{"Name":"user-circle-16","Size":16},{"Name":"user-circle-fill-24","Size":24},{"Name":"user-circle-fill-16","Size":16},{"Name":"user-minus-24","Size":24},{"Name":"user-minus-16","Size":16},{"Name":"user-plus-24","Size":24},{"Name":"user-plus-16","Size":16},{"Name":"user-x-24","Size":24},{"Name":"user-x-16","Size":16},{"Name":"users-24","Size":24},{"Name":"users-16","Size":16},{"Name":"vault-24","Size":24},{"Name":"vault-16","Size":16},{"Name":"verified-24","Size":24},{"Name":"verified-16","Size":16},{"Name":"video-24","Size":24},{"Name":"video-16","Size":16},{"Name":"video-off-24","Size":24},{"Name":"video-off-16","Size":16},{"Name":"volume-24","Size":24},{"Name":"volume-16","Size":16},{"Name":"volume-down-24","Size":24},{"Name":"volume-down-16","Size":16},{"Name":"volume-2-24","Size":24},{"Name":"volume-2-16","Size":16},{"Name":"volume-x-24","Size":24},{"Name":"volume-x-16","Size":16},{"Name":"wall-24","Size":24},{"Name":"wall-16","Size":16},{"Name":"watch-24","Size":24},{"Name":"watch-16","Size":16},{"Name":"webhook-24","Size":24},{"Name":"webhook-16","Size":16},{"Name":"wifi-24","Size":24},{"Name":"wifi-16","Size":16},{"Name":"wifi-off-24","Size":24},{"Name":"wifi-off-16","Size":16},{"Name":"wrench-24","Size":24},{"Name":"wrench-16","Size":16},{"Name":"x-24","Size":24},{"Name":"x-16","Size":16},{"Name":"x-circle-24","Size":24},{"Name":"x-circle-16","Size":16},{"Name":"x-circle-fill-24","Size":24},{"Name":"x-circle-fill-16","Size":16},{"Name":"x-diamond-24","Size":24},{"Name":"x-diamond-16","Size":16},{"Name":"x-diamond-fill-24","Size":24},{"Name":"x-diamond-fill-16","Size":16},{"Name":"x-hexagon-24","Size":24},{"Name":"x-hexagon-16","Size":16},{"Name":"x-hexagon-fill-24","Size":24},{"Name":"x-hexagon-fill-16","Size":16},{"Name":"x-square-24","Size":24},{"Name":"x-square-16","Size":16},{"Name":"x-square-fill-24","Size":24},{"Name":"x-square-fill-16","Size":16},{"Name":"zap-24","Size":24},{"Name":"zap-16","Size":16},{"Name":"zap-off-24","Size":24},{"Name":"zap-off-16","Size":16},{"Name":"zoom-in-24","Size":24},{"Name":"zoom-in-16","Size":16},{"Name":"zoom-out-24","Size":24},{"Name":"zoom-out-16","Size":16}]
```

Comment:
(blocking) please document this change as requested. i followed your instructions in https://github.com/hashicorp/flight/pull/79#issuecomment-902212675 on the other PR https://github.com/hashicorp/flight/pull/88 and it didn't work for me. i assume you used regex.

Reply:
@amyrlam This is the file straight from the export, I replaced the one where I'd made changes. So this is the raw file with no changes.

Reply:
gotcha, thanks for the explanation, that wasn't clear to me as a reviewer that you made that change